src/monocle-ts/ISOType.ts

```diff
@@ -0,0 +1,5 @@
+import { Iso } from 'monocle-ts'
+import * as t from 'io-ts'
+
+export const ISOType = <I, O>(name: string, iso: Iso<I, O>, is: (x: O) => x is O): t.Type<I, O> =>
```

Comment:
Can we name this function `typeFromIso` as you proposed, or even better just `fromIso`?
The signature of `is` should be `is: (x: any) => x is O`...

Reply:
oh yes, my bad